Mathlib/AlgebraicGeometry/EllipticCurve/Group.lean

```diff
@@ -525,6 +541,7 @@ lemma degree_norm_ne_one [IsDomain R] (x : W.CoordinateRing) :
   exact (lt_max_of_lt_right <| by exact (cmp_eq_lt_iff ..).mp rfl).ne'
 #align weierstrass_curve.coordinate_ring.degree_norm_ne_one WeierstrassCurve.Affine.CoordinateRing.degree_norm_ne_one
 
+variable {W} in
```

Comment:
Same here: why this change?

Reply:
I guess this one preserves the total number of lines :)